admin_manual/maintenance/migrating.rst

```diff
@@ -2,199 +2,235 @@
+   This ensures, should anything unforeseen happen, that you can go back to your existing installation and resume availability of your |name| installation while debugging the problem.
 
-#.  On the original machine turn on maintenance mode and then stop ownCloud. After waiting for 6-7 minutes for all sync clients to register the server as in maintenance mode, stop the application and/or Web server that serves ownCloud. (See :ref:`maintenance_commands_label`.)
+How to Migrate
+--------------
 
-#.  Create a dump from the database and copy it to the new machine, and import it into the new database (See :ref:`backup_database_label` and :ref:`restore_database_label`).
+Firstly, set up the new machine with your desired Linux distribution. 
+At this point you can either :doc:`install ownCloud manually <../installation/source_installation>` via the compressed archive, or doc:`with your Linux package manager <../installation/linux_installation>`.
 
-#.  Copy ONLY your data, configuration and database files from your original ownCloud instance to the new machine (See :doc:`backup` and :ref:`restore_directories_label`). You must keep the ``data/`` directory's original filepath. Please, **do not change this!**
+Then, on the original machine turn on maintenance mode and then stop ownCloud. 
+After waiting for 6 - 7 minutes for all sync clients to register that the server is in maintenance mode, ref:`stop the web server <maintenance_commands_label>` that is serving ownCloud.
 
-#. The data files should keep their original timestamp (can be done by using ``rsync`` with ``-t`` option) otherwise the clients will re-download all the files after the migration. This step might take several hours, depending on your installation.
+After that, :ref:`create a database dump from the database <backup_database_label>`, copy it to the new machine, and :ref:`import it into the new database <restore_database_label>`.
 
-#.  With ownCloud still in maintenance mode (confirm!) and **BEFORE** changing the ``CNAME`` record in the DNS start up the database, Web server / application server on the new machine and point your Web browser to the migrated ownCloud instance. Confirm that you see the maintenance mode notice, that a logfile entry is written by both the Web server and ownCloud and that no error messages occur. Then take ownCloud out of maintenance mode and repeat. Log in as admin and confirm normal function of ownCloud.
+Then, copy only your data, configuration, and database files from your original ownCloud instance to the new machine (See :doc:`backup` and :ref:`restore_directories_label`). 
 
-#.  Change the ``CNAME`` entry in the DNS to point your users to the new
-    location.
-    
-With the ``CNAME`` updated, you now need to updated the trusted domains.
+.. warning::
+   You must keep the ``data/`` directory's original filepath. Please, do not change this!
```

Comment:
Here we're advising this shouldn't be changed, below its noted that this can be done later:

https://github.com/owncloud/documentation/pull/3276/files#diff-8b225109e5ecb56dbb9dced5cd16b35fR186